concepts/80 Troubleshooting/45 Component Content is not Scrollable/10 The Page is Scrollable, the Component is Not.md

```diff
@@ -0,0 +1,6 @@
+If your component is not scrollable, while you can scroll the page, check the following conditions:
```

Comment:
I'm not sure what you mean by "not scrollable". The description is very similar to the previous point (I already mentioned that). But can we also be more specific about "not scrollable"? Does it not display a scrollbar? It displays a scrollbar, but that scrollbar doesn't respond to user actions? It already displays all rows, so doesn't need a scrollbar?